database-plugins/src/test/java/io/cdap/plugin/db/batch/source/DBSourceTestRun.java

```diff
@@ -456,4 +457,18 @@ public void testNonExistentDBTable() throws Exception {
     assertDeploymentFailure(appId, etlConfig, "ETL Application with DB Source should have failed because of a " +
       "non-existent source database.");
   }
+
+  @Test
+  public void testFQNGeneration() throws Exception {
```

Comment:
Each test should be testing a single functionality. Let's separate the tests for each of the DBs